package.json

```diff
@@ -13,12 +13,12 @@
         "build": "tsc -p tsconfig.json",
         "db:migrate": "./node_modules/.bin/typeorm migration:run --config ./lib/src/ormconfig.js ",
         "db:revert": "./node_modules/.bin/typeorm migration:revert --config ./lib/src/ormconfig.js ",
-        "integration-test": "mocha --require source-map-support/register --require make-promises-safe lib/integration-test/**/*_test.js --timeout 200000 --exit",
-        "test": "yarn test:rest && yarn test:metatx && yarn test:rfqt && yarn test:rfqm",
-        "test:metatx": "env-cmd -f ./test/test_env mocha --require source-map-support/register --require make-promises-safe lib/test/meta_transaction_**.js --timeout 200000 --exit --bail",
-        "test:rest": "env-cmd -f ./test/test_env mocha --exclude lib/test/meta_transaction_test.js --exclude lib/test/rfqt_test.js --exclude lib/test/rfqm_test.js --require source-map-support/register --require make-promises-safe lib/test/**/*_test.js lib/test/*_test.js --timeout 200000 --exit --bail",
-        "test:rfqt": "env-cmd -f ./test/test_env mocha --require source-map-support/register --require make-promises-safe lib/test/rfqt_**.js --timeout 200000 --exit --bail",
-        "test:rfqm": "env-cmd -f ./test/test_env mocha --require source-map-support/register --require make-promises-safe lib/test/rfqm_**.js --timeout 200000 --exit --bail",
+        "integration-test": "mocha --require source-map-support/register --require make-promises-safe 'lib/integration-test/**/*_test.js' --timeout 200000 --exit",
+        "test": "yarn test:rest && yarn test:metatx && yarn test:rfqt",
+        "test:metatx": "env-cmd -f ./test/test_env mocha --require source-map-support/register --require make-promises-safe 'lib/test/meta_transaction_**.js' --timeout 200000 --exit --bail",
+        "test:rest": "env-cmd -f ./test/test_env mocha --exclude lib/test/rfqt_test.js --exclude lib/test/meta_transaction_test.js --require source-map-support/register --require make-promises-safe 'lib/test/**/*_test.js' --timeout 200000 --exit --bail",
```

Comment:
without the `''` the glob was only matching 1 test (rfqm), this has since been moved to its own command but we should still fix the matching to avoid future issues :)